src/utils/nft-universal-links.ts

```diff
@@ -0,0 +1,39 @@
+import { transform } from 'lodash-es';
+
+import { UNIVERSAL_LINKS_DOMAIN_URI_PREFIX } from './env.utils';
+import { isTruthy } from './is-truthy';
+import { getTempleUniversalLink } from './universal-links';
+import { buildSafeURL } from './url.utils';
+
+export const buildCollectibleUniversalLink = (slug: string, image?: string, title?: string, description?: string) =>
+  getTempleUniversalLink(
+    '/nft',
+    transform<string | undefined, StringRecord>(
+      { slug, image, title, description },
+      (acc, value, key) => {
+        if (isTruthy(value) && key === 'description') {
+          acc[key] = value.length > 500 ? `${value.slice(0, 500)}...` : value;
+        } else if (isTruthy(value)) {
+          acc[key] = value;
+        }
+
+        return acc;
+      },
+      {}
+    )
```

Comment:
We have to delete keys undefined values to prevent passing invalid query parameters. In the extract from `node` console below, it is shown what may happen if we do without it:
```
> new URLSearchParams({ p1: 'p1', p2: undefined }).toString()
'p1=p1&p2=undefined'
```